pkg/testutil/golden/golden.go

```diff
@@ -0,0 +1,28 @@
+// Package golden provides function and helpers to use golden file for
+// testing purpose.
+package golden
+
+import (
+	"flag"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+)
+
+var update = flag.Bool("test.update", false, "update golden file")
```

Comment:
I think it's kind of bad form to register CLI flags from arbitrary non-main packages. 

I really like the idea of being able to update these files easily, but I wonder if we could do this with a separate tool